samples/dashboards/demo/personal-portfolio/demo.js

```diff
@@ -0,0 +1,429 @@
+const basicInvestmentPlan = {
+    interval: 30, // Every 30 days
+    amount: 200   // Amount in EUR
+};
+
+
+const stockCollection = [
+    {
+        tradingSymbol: 'NFLX',
+        ISIN: 'US64110L1061',
+        SecID: '0P000003UP'
+    }, {
+        tradingSymbol: 'MSFT',
+        ISIN: 'US5949181045',
+        SecID: '0P000003MH'
+    }, {
+        tradingSymbol: 'AMZN',
+        ISIN: 'US0231351067',
+        SecID: '0P000000B7'
+    }, {
+        tradingSymbol: 'GOOGL',
+        ISIN: 'US02079K3059',
+        SecID: '0P000002HD'
+    }
+];
```

Comment:
Keep consistent brackets formatting:
```suggestion
const stockCollection = [{
    tradingSymbol: 'NFLX',
    ISIN: 'US64110L1061',
    SecID: '0P000003UP'
}, {
    tradingSymbol: 'MSFT',
    ISIN: 'US5949181045',
    SecID: '0P000003MH'
}, {
    tradingSymbol: 'AMZN',
    ISIN: 'US0231351067',
    SecID: '0P000000B7'
}, {
    tradingSymbol: 'GOOGL',
    ISIN: 'US02079K3059',
    SecID: '0P000002HD'
}];
```